packages/vm/src/evm/evm.ts

```diff
@@ -153,6 +153,7 @@ export default class EVM {
 
     const oldRefund = this._refund.clone()
 
+    // TODO: checkpoint needs to take into account transient storage
```

Comment:
Yep, good point, this should probably be added in `StateManager`, we cannot directly use the storage logic since this is saved into a `Map` and not to disk / temporary to memory.

Reply:
I think the current logic in `transientStorage` would make this work, right?

Reply:
You are correct, this is a dead comment now